foyer/forcefield.py

```diff
@@ -308,6 +308,15 @@ def apply(self, topology, references_file=None, *args, **kwargs):
         references_file : str, optional, default=None
             Name of file where force field references will be written (in Bibtex
             format)
+        use_residue_map : boolean
+            Store atomtyped topologies of residues to a dictionary that maps
+            them to residue names.  Each topology, including atomtypes, will be
+            copied to other residues with the same name. This avoids repeatedly
+            calling the subgraph isomorphism on idential residues and should
```

Comment:
Typo (should be `identical`)

Reply:
I should probably have a vim plugin that prevents me from making these sorts of mistakes....